practical-unit-testing-with-rtl/Exercise1-queries/problem/src/App.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+
+const Button = () => {
+  return <button> Click me to submit!</button>;
```

Comment:
Right now, clicking this button will refresh the page. I think you can avoid this by adding `type="button"`.